libkbfs/md_journal.go

```diff
@@ -935,7 +978,8 @@ func (e MDJournalConflictError) Error() string {
 // rmd becomes the initial entry.
 func (j *mdJournal) put(
 	ctx context.Context, signer kbfscrypto.Signer,
-	ekg encryptionKeyGetter, bsplit BlockSplitter, rmd *RootMetadata) (
+	ekg encryptionKeyGetter, bsplit BlockSplitter, rmd *RootMetadata,
+	isLocalSquash bool) (
```

Comment:
There are some invariants you can check if `isLocalSquash` is true, right? In particular, BID should be NullBranchID, and all previous MDs must have IsLocalSquash also set (probably suffices to just check the most recent one, if any).